app/Resources/views/page.html.twig

```diff
@@ -117,14 +117,25 @@
             <meta name="description" content="{{ description|striptags|single_line|trim }}">
         {% endif %}
 
-        {% if contentHeader is defined and contentHeader.image is defined %}
+        {% if item is defined and item.banner is defined and item.banner %}
+            <meta name="twitter:card" content="summary_large_image">
+            <meta property="og:image" content="{{ iiif_uri(item.banner) }}">
+            <meta property="og:image:width" content="{{ item.banner.width }}">
+            <meta property="og:image:height" content="{{ item.banner.height }}">
```

Comment:
Currently using the source width/height. This could potentially be too big for Facebook (8MB), Twitter (4096x4096, 5MB) etc.

Reply:
There are some examples where the sources are >10MB (sometimes JPG), but somehow the served full images become 2-3 MB instead:
https://iiif.elifesciences.org/digests%3A37621%2Fdigest-37621.jpg/full/full/0/default.jpg
https://iiif.elifesciences.org/digests/37349/digest-37349.png/full/full/0/default.jpg
https://iiif.elifesciences.org/digests/39468/digest-39468.png/full/full/0/default.jpg
https://iiif.elifesciences.org/digests/35066/digest-35066.jpg/full/full/0/default.jpg

Reply:
Will add in a cap.